python/triton/compiler/compiler.py

```diff
@@ -236,6 +236,15 @@ def filter_traceback(e: BaseException):
         e.__traceback__ = frames[0]
 
 
+def triton_spirv_dump(data, file_name, ext):
```

Comment:
```suggestion
def triton_spirv_dump(data, file_name):
    ext = "spv"
```
Isn't it always `"spv"`?